dev/io.openliberty.microprofile.health.file.healthcheck.internal_fat/fat/src/io/openliberty/microprofile/health/file/healthcheck/fat/MPConfigDefaultValuesTest.java

```diff
@@ -0,0 +1,397 @@
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.microprofile.health.file.healthcheck.fat.utils.Constants;
+import io.openliberty.microprofile.health.file.healthcheck.fat.utils.HealthFileUtils;
+import io.openliberty.microprofile.health.internal_fat.shared.HealthActions;
+
+/**
+ *
+ */
+@RunWith(FATRunner.class)
+public class MPConfigDefaultValuesTest {
+
+    final static String DEFAULT_SERVER = "HealthServer";
+    final static String MPCONFIG_DEFAULT_STARTUP_UP_SERVER = "DefaultStartupUpServer";
+    final static String MPCONFIG_DEFAULT_READINESS_UP_SERVER = "DefaultReadinessUpServer";
+    final static String MPCONFIG_DEFAULT_ALL_UP_SERVER = "DefaultAllUpServer";
+
+    final static String REGULAR_APP = "RegularApp";
+    final static String REGULAR_APP_WAR = REGULAR_APP + ".war";
+
+    final static String DELAYED_APP = "DelayedApp";
+    final static String DELAYED_APP_WAR = DELAYED_APP + ".war";
+
+    private static final String[] FAILS_TO_START_EXPECTED_FAILURES = { "CWMMH0052W", "CWMMH0054W", "CWMMH0053W" };
+
+    public static final int APP_STARTUP_TIMEOUT = 120 * 1000;
+
+    private static enum HealthCheck {
+        LIVE, READY, STARTED, HEALTH;
+    }
+
+    private static enum Status {
+        SUCCESS, FAILURE;
+    }
+
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(FeatureReplacementAction.ALL_SERVERS,
+                                                             MicroProfileActions.MP61, // mpHealth-4.0 w/ EE9
+                                                             MicroProfileActions.MP70_EE10, // mpHealth-4.0 FULL EE10
+                                                             MicroProfileActions.MP70_EE11, // mpHealth-4.0 FULL EE11
+                                                             HealthActions.MP14_MPHEALTH40, // mpHealth-4.0 FULL EE7
+                                                             HealthActions.MP41_MPHEALTH40); //mpHealth-4.0 FULL EE8
+
+    @Server(DEFAULT_SERVER)
+    public static LibertyServer defaultServer;
+
+    @Server(MPCONFIG_DEFAULT_STARTUP_UP_SERVER)
+    public static LibertyServer mpConfigDefaultStartupUpServer;
+
+    @Server(MPCONFIG_DEFAULT_READINESS_UP_SERVER)
+    public static LibertyServer mpConfigDefaultReadinessUpServer;
+
+    @Server(MPCONFIG_DEFAULT_ALL_UP_SERVER)
+    public static LibertyServer mpConfigAllUpServer;
+
+    @BeforeClass
+    public static void beforeClass() {
+
+    }
+
+    public void cleanup(LibertyServer server) throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer(FAILS_TO_START_EXPECTED_FAILURES);
+        }
+    }
+
+    @Test
+    public void NoDefaultConfigTest() throws Exception {
+        final String METHOD_NAME = "NoDefaultConfigTest";
+        LibertyServer server = defaultServer;
+
+        WebArchive regularWAR = ShrinkWrap
```

Comment:
Wouldn't it be cleaner if you put the deployment of the regular and delay apps in a common method, and call that method for each test case, instead of repeating the same code, similar to cleanup?